kairon/actions/definitions/custom_parallel_actions.py

```
@@ -135,69 +134,39 @@
-        request_json = action_instance
+        request_json = action_instance.copy()
         request_json['next_action'] = action_name
+        request_json['trigger_info'] = {
+            "trigger_name": self.name,
+            "trigger_type": ActionType.parallel_action.value
+        }
 
         url = Utility.environment["action"].get("url")
 
         async with aiohttp.ClientSession() as session:
             async with session.post(url, json=request_json) as response:
```

Ruff (0.11.9)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)